pkg/aws_keyhub/aws.go

```diff
@@ -77,7 +77,7 @@ func WriteCredentialFile(credentials *sts.Credentials) {
 		logrus.Fatal("Failed to read credentials file:", err)
 	}
 
-	sec := cfg.Section("keyhub") // Auto-create if not exists
+	sec := cfg.Section(awsProfile) // Auto-create if not exists
```

Comment:
I would prefer to add a new debug log line just before or after `cfg.SaveTo` so that it is clear to what credential file under what cfg.Section (profile name) the credentials where written. The actual credentials shouldn't be logged. Would you like to add this?

Reply:
Added! Also moved both log statements to below the saveTo, which seems logical as if the `cfg.SaveTo`
would fail the log lines would not be valid, this might help users spot something went wrong.